src/Map.jsx

```diff
@@ -166,7 +207,7 @@ class Map extends Component {
       .filter(route => selectedRouteNames.has(route.rid))
       .reduce((layers, route) => [
         ...layers,
-        this.state.showStops ? getStopMarkersLayer(route) : null,
+        this.state.showStops ? getStopMarkersLayer(route, i => this.getStopInfo(route, i)) : null,
```

Comment:
Would it be more clear if we did `getStopMarkersLayer(route, markerClick => this.getStopInfo(route, markerClick.object.position))` and then changed `getStopInfo`'s signature to  `getStopInfo(route, stopLocation)`, where `stopLocation` is of type `{lat: float, lon: float}`? (I just want it to be really easy for us to add Flow typing eventually)

Reply:
@EddyIonescu I can do all that, but I might have to change the naming. We have an ESLint that limits you to only 100 characters per line

Reply:
@ckrajewski couldn't we use multiple lines? Didn't know it was limited to 100, maybe I should change it to 90 :)

Reply:
@EddyIonescu there's another lint module that prevents new lines/whitespace function-paren-newline

Reply:
I see. Couldn't we do
```
this.state.showStops
   ? stuff
   : other stuff
```